test/sender.buffer.test.ts

```diff
@@ -167,6 +167,245 @@ describe("Sender message builder test suite (anything not covered in client inte
     await sender.close();
   });
 
```

Comment:
Would be great to add a few integration tests to verify that arrays are successfully inserted into the table.

Reply:
seems that 1D empty array works fine, multi-dimensional empty arrays not entirely.
probably a server bug.
added support for empty arrays in https://github.com/questdb/nodejs-questdb-client/pull/50/commits/c4feb7b4283290d3680991da88e4db8c48b0ae30

also added integration tests